### PR TITLE
feat: lobby

### PR DESCRIPTION
**Pull request by izabellalarsson**
- User redirects to lobby if localstorage item is'nt set.
- User fill in display name 
- Name is shown when user is sending a message

### CODE REVIEW

File: packages/client/src/routes/Room/Room.tsx [hunk 1]
```diff
@@ -15,9 +14,30 @@ const Header = styled.div`
...
   const [message, setMessage] = React.useState("")
   const { roomId } = useParams()
+  const navigate = useNavigate()
+  const [chatMember, setChatMember] = React.useState<ChatMember>({
+    name: "",
+    id: "",
+  })
+
+  React.useEffect(() => {
+    if (!localStorage.getItem("member_id")) {
```
Comment (dubbelnisse) [inline packages/client/src/routes/Room/Room.tsx]: I would set the localstorage calls as variables for increased readability and one of them is ised multiple times

Comment (momentiris) [reply packages/client/src/routes/Room/Room.tsx]: Agree. Maybe make a utils file? localStorage.setItem can actually throw an error too, so wrap it in a try/catch. I think there is code that does this in gavagai, for reference :)

Comment (izabellalarsson) [reply packages/client/src/routes/Room/Room.tsx]: I will look in to that!

**Comment by izabellalarsson** [comment]
Moved the localstorage calls to a utils file for cleaner code.

**Comment by momentiris** [comment]
Nice. I think localstorage.getItem doesnt actually throw, so that one doesn't need the try/catch. but lets not care about that for now. I also suspect that reference for this code is in gavagai.. which i wrote ;)

**Pull request merged**